docker/dockerbuild.sh

```diff
@@ -2,7 +2,7 @@
 set -euxo pipefail
 
 # Install programs required for build
-npm install -g yarn bower
+npm install -g bower
```

Comment:
Yarn is already installed in the `node:8` image; explicitly installing it broke the build.